TradeItIosTicketSDK2/NumberFormatter.swift

```diff
@@ -2,22 +2,27 @@ class NumberFormatter: NSObject {
     private static let currencyFormatter = Foundation.NumberFormatter()
     private static let quantityFormatter = Foundation.NumberFormatter()
     
-    static func formatCurrency(_ number: NSNumber, minimumFractionDigits: Int = 2, maximumFractionDigits: Int = 2, displayVariance: Bool = false, currencyCode: String? = TradeItPresenter.DEFAULT_CURRENCY_CODE) -> String {
+    static func formatCurrency(
+        _ number: NSNumber,
+        minimumFractionDigits: Int = 2,
+        maximumFractionDigits: Int = 2,
+        displayVariance: Bool = false,
+        currencyCode: String? = TradeItPresenter.DEFAULT_CURRENCY_CODE
+    ) -> String {
         currencyFormatter.numberStyle = .currency
         currencyFormatter.currencyCode = currencyCode
         currencyFormatter.currencySymbol = overrideCurrencySymbol(forCurrencyCode: currencyCode)
-        if displayVariance {
-            currencyFormatter.positivePrefix = "+" + currencyFormatter.currencySymbol
-            currencyFormatter.negativePrefix = "-" + currencyFormatter.currencySymbol
-        } else {
-            currencyFormatter.positivePrefix = nil
-            currencyFormatter.negativePrefix = nil
-        }
         currencyFormatter.minimumFractionDigits = minimumFractionDigits
         currencyFormatter.maximumFractionDigits = maximumFractionDigits
-        return currencyFormatter.string(from: number) ?? TradeItPresenter.MISSING_DATA_PLACEHOLDER
+        var formattedCurrency = currencyFormatter.string(from: number)
+        if displayVariance,
+            number.doubleValue > 0.0,
+            let unwrappedFormattedCurrency = formattedCurrency {
+            formattedCurrency = "+\(unwrappedFormattedCurrency)"
```

Comment:
`NumberFormatter#string()` auto prepends `-` out of the box. Presumably we _always_ want to display the `-` sign for negative numbers -- so we need only account for `+`. 

`NumberFormatter` provides some config options (`negativePrefix`, `negativeFormat`, ...), however, they all come with quirks (not respect locale formatting for currencies, and whatever else Apple shoves into their black box.)